src/App.test.js

```diff
@@ -85,7 +85,7 @@ describe('App', () => {
 });
 
   
-  describe.skip('Wave 3:  Winner tests', () => {
+  describe('Wave 3:  Winner tests', () => {
```

Comment:
The winner tests (as well as any other test checking for x/o) would all need to be updated to look for 🍿/🍭.